### ISSUE TITLE
Full Accessibility & 508 Office Audit Decision Tools - Find a VA Form Post-MVP Enhancements

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#18336]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] VFS Product Manager: create issue and fill in feature name in the title and other bolded information appropriately
- [x] VFS: Link to this issue once created in [#vfs-platform-support](https://dsva.slack.com/channels/vfs-platform-support) in Slack; tag @ Trevor Pierce
- [ ] VSP: Accessibility specialists create sub-issues attached to this epic with findings
- [ ] VFS: Coordinate with the VA 508 Office to address accessibility findings
- [ ] VSP: [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/u/1/d/1d219oL1zCvCvnv1Bx-dI-GMzwgbarLv9_bzMSa3ULjA/edit#gid=1341642809) is updated

## Artifacts - _please provide the following artifacts when you request a full review_
- Access information for the tool or feature in staging _**do not put staging credentials in your `va.gov-team` ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository**_
- Link to verification that accessibility checks are complete:
  - (preferred) Link to your [TestRail VSP accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc).
OR
  - Link to a doc in your product folder that indicates all required checks from the accessibility [staging review prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) doc are complete.
- List of known issues (with links to tickets when applicable)

## Additional Notes
As part of this review, VSP will send your product to VA's 508 office for approval (as applicable)

**Comment by MarciMcGuireGCIO** [comment]
Moving all Find a VA Form 508 issues into a collective Epic to clean up our backlog.

**Comment by shiragoodman** [comment]
hey @joshkimux  - not sure if you're aware of this one. adding you as an assignee.

**Comment by joshkimux** [comment]
Thank you!!

**Comment by MarciMcGuireGCIO** [comment]
Given the updated VSP guidance and the fact we have outstanding 508 issues from Staging, I am withdrawing this request and will re-submit once we have addressed those issues.

**Comment by MarciMcGuireGCIO** [comment]
Closing this as there is already a pending full 508 review from the MVP.
